requirements.txt

```diff
@@ -1,7 +1,2 @@
-matplotlib>=1.3.1
-numpy>=1.11
-scipy>=0.17.0
-six>=1.7.3
-networkx>=1.8
-pillow>=2.1.0
-PyWavelets>=0.4.0
+-r requirements/default.txt
```

Comment:
Why keep this file?

Reply:
I think this is the default that everyone tries to install, so not a bad idea to have it.